### DISCUSSION TITLE
PyCUDA with sparse iterative linear system solver

### DISCUSSION THREAD

**Discussion by HongRayChen** [inducer/pycuda#421] [General]
I previously used CuPy to accelerate sparse iterative linear equation solving for FEA but we have been experiencing some difficulties in installing CuPy on computers. 
I saw Professor Kloeckner had an iterative CUDA on his [webpage](https://mathema.tician.de/software/iterative-cuda/) and mentioned that a more flexible version is available in PyCUDA as well. 
I am pretty new to CUDA and PyCUDA. I wonder if anyone can point me to some resources or maybe some project that uses PyCUDA and the sparse iterative solver?

**Comment by inducer** [comment]
This here:

https://github.com/inducer/pycuda/blob/939675adc16742861a533249869d7ab93bc305c8/examples/from-wiki/sparse_solve.py

might be a starting point.
